docs/conf.py

```diff
@@ -147,5 +147,34 @@
 
 # qiskit package specific variables
 html_context = {
-    'analytics_enabled': True
+    # toggle analytics
+    'analytics_enabled': True,
+
+    # dummy list for versions
+    'version_list': [0.1, 0.2, 0.3],
+
+    # example custom sidebar headings & subheadings
+    'sidebar_headings': [
+        {
+            'title': 'Custom Heading 1 (no subheadings)',
+            'url': 'sphinx_guide/images', # use sphinx toctree
+        },
+        {
+            'title': 'Custom Heading 2 (with subheadings)',
+            'subheadings': [
+                {
+                    'title': 'Custom Subtitle 1 (external link)',
+                    'url': 'https://google.com', # use external url
```

Comment:
Can you change the external link to something else? Like https://qiskit.org/ or youtube.com/qiskit